### ISSUE TITLE
Any Example?

### ISSUE THREAD

**Issue by Carbowix** [ignu/trump-facts#2]
Do you have any example for your module?

because i wanna know how we can use that

**Comment by ignu** [comment]
I haven't kept this up to date since the election, but [DonaldJTrumpis.com](http://donaldjtrumpis.com/) was a consumer. Repo for that is here [ignu/trump-facts-web-client: web consumer of trump-facts](https://github.com/ignu/trump-facts-web-client)

**Comment by Carbowix** [comment]
Yes ik about those, but i want a Node JS example about how to use your Node Module.
Does it consists of Searching or what. Or it just gets a random thing. Example makes everything coolers.

Thanks, In Advance

**Comment by ignu** [comment]
It's just a javascript object https://github.com/ignu/trump-facts/blob/master/data.js

My thought about breaking it out on github would be to make it an easy way to crowdsource data.
